packages/runtime/test/src/parser/parser.ts

```diff
@@ -2041,6 +2048,95 @@ describe("Parser", function () {
 					RUNTIME_ERRORS.TEAL.ASSERT_LENGTH
 				);
 			});
+
+			it("gitxn", () => {
```

Comment:
I think we should change all of this to something like 

```suggestion
			it("Should throw and error when <specific case>", () => {
```

Reply:
I updated test cases more sense

Reply:
> I think we should change all of this to something like

I updated test cases more sense